public/app/features/admin/state/apis.ts

```diff
@@ -31,3 +33,64 @@ export const getServerStats = async (): Promise<ServerStat[]> => {
     throw error;
   }
 };
+
+export const getLdapState = async (): Promise<LdapConnectionInfo> => {
+  return await getBackendSrv().get(`/api/admin/ldap/status`);
+};
+
+export const getLdapSyncStatus = async (): Promise<SyncInfo> => {
```

Comment:
Seems like `ldap-sync-status`, `ldap/sync/${userId}` are missing from this PR and so this does not work without enterprise.

Reply:
`ldap/sync/:userId` was just merged in #18976. `/ldap-sync-status` is a part of the enterprise, you're right. I have to fix it.